src/components/Objects/Card/CardTagContainer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function CardTagContainer({ children }) {
```

Comment:
Some of the card components seem to be the almost exact some code, is it possible or worth it to consolidate?

Reply:
For me it was consolidating and informing with the HTML/JSX I can see the argument for getting rid of this CardTagContainer if the projects card tag container has different functionality